src/analysis/retail/shaman/restoration/normalizers/CastLinkNormalizer.ts

```diff
@@ -325,6 +328,42 @@ const EVENT_LINKS: EventLink[] = [
       return c.hasTalent(talents.WHIRLING_ELEMENTS_TALENT);
     },
   },
+  // Lively Totems : When you summon a Healing Tide Totem, Healing Stream Totem, Cloudburst Totem, Mana Tide Totem, or Spirit Link Totem you cast a free instant Chain Heal at 100% effectiveness.
+  {
```

Comment:
would be good to add a `GlobalCooldown` override that prevents these casts from triggering the GCD. i don't think the uptime stats are currently being used by rsham, though, so it isn't high priority

Reply:
Do you have any examples of this? I wasn't aware of this functionality